mapviz_plugins/src/odometry_plugin.cpp

```diff
@@ -208,6 +193,21 @@ namespace mapviz_plugins
         }
       }
     }
+
+    if (points_.empty() ||
+        stamped_point.point.distance(points_.back().point) >=
```

Comment:
In complex conditionals like this, prefer to explicitly using parentheses instead of relying on order of operations. Other coders may not necessarily know the order of operations, and omitting the parentheses forces readers to parse this entire expression before understanding its meaning. Optimize code for reading.

Please surround the `>=` expression with parentheses.